packages/shared/src/styles/constants.scss

```diff
@@ -196,3 +196,8 @@ $gradient-webmoney: linear-gradient(90deg, #1a77ac 0%, #0068a3 100%);
 /* App Cards gradient background */
 $gradient-virtual: linear-gradient(274.25deg, #333333 9.01%, #5c5b5b 103.31%);
 $gradient-virtual-swap-free: linear-gradient(58.51deg, #061828 28.06%, #1a3c60 93.51%);
+
+/* Wallets */
+$wallets-banner-box-shadow: 0px 12px 16px -4px rgba(14, 14, 14, 0.08), 0px 4px 6px -2px rgba(14, 14, 14, 0.03);
+$wallets-banner-border-radius-desktop: 16px;
```

Comment:
we have `$BORDER_RADIUS: 4px; ` here, let's define these general  `BORDER_RADIUS_L` and `BORDER_RADIUS_XL`  radiuses for these 2 new options and get benefit of them in whole the code base and not just for banners.

Reply:
we can use it like this:
`$BORDER_RADIUS * 4` and `$BORDER_RADIUS * 2`
instead of defining new values.